packages/worksheet/src/pages/CreateWorksheet.js

```diff
@@ -16,25 +16,87 @@ import {
   Actionsheet,
   Box,
   Pressable,
-  Checkbox,
   VStack,
-  Center,
-  ScrollView,
   FormControl,
   Input,
+  ScrollView,
 } from "native-base";
 import React from "react";
 import { useTranslation } from "react-i18next";
 import { getAllQuestions } from "services";
 import manifest from "../manifest.json";
 
+const defaultInputs = [
```

Comment:
Move this to a new file. Refactor this into a smaller file. No more than 150 lines.

Reply:
Refactor all large files to smaller files.

Reply:
I am already made this in worksheet create flow PR